tests/scripts/thread-cert/config.py

```diff
@@ -46,6 +46,9 @@
 LINK_LOCAL = 'fe80:/112'
 LINK_LOCAL_REGEX_PATTERN = '^fe80:.*'
 ALOC_FLAG_REGEX_PATTERN = '.*:fc..$'
+LINK_LOCAL_MULTICAST_ADDRESS = 'ff32:40:fdde:ad00:beef:0:0:1'
```

Comment:
suggestion:
rename `LINK_LOCAL_MULTICAST_ADDRESS` to `LINK_LOCAL_All_THREAD_NODES_MULTICAST_ADDRESS`, making the name itself more descriptive.